opencontrail/map.jinja

```diff
@@ -79,7 +84,7 @@ database:
 web:
   Debian:
     pkgs:
-         ['contrail-web-controller', 'nodejs-legacy', 'gettext-base']
+         ['contrail-web-controller', 'nodejs', 'gettext-base']
```

Comment:
Yes, nodejs was in conflict with nodejs, which was already installed in the time the nodejs-legacy package was being installed. It's probably installed by some dependent package.